source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -709,11 +710,33 @@ function get_adjacent_handbook_post_link( $output, $format, $link, $post, $adjac
 	$title = apply_filters( 'the_title', $post->post_title, $post->ID );
 	$url   = get_permalink( $post );
 
+	$screen_reader_content = sprintf(
+		$is_previous
+			? /* translators: %s: post title */
+			__( 'Previous chapter: %s', 'wporg' )
+			: /* translators: %s: post title */
+			__( 'Next chapter: %s', 'wporg' ),
+		$title
+	);
+
+	$content = str_replace(
+		'%title',
+		sprintf(
+			'<span aria-hidden="true" class="post-navigation-link__label">%1$s</span>
+			<span aria-hidden="true" class="post-navigation-link__title">%2$s</span>
```

Comment:
Adding these classes, which would usually only be applied if the `wp:post-navigation-link` had a label configured, means that the existing styles from the parent theme continue to work:

![Screenshot 2023-11-22 at 11 16 34 AM](https://github.com/WordPress/wporg-developer/assets/1017872/3b20257a-cfbb-4090-9b8e-dee7a02a437c)